src/inspector/inspector_tcp.cpp

```diff
@@ -88,8 +100,7 @@ void tcp_connection::read_loop_async() {
   });
 }
 
-// !!COPY
-void tcp_connection::write_async(std::vector<char> message_) {
+void tcp_connection::write_async(const std::vector<char>& message_) {

```

Comment:
>r>& m [](start = 54, length = 5)

Why not just have Api take &&?

It's being moved anyways on line 107
Also, why isn't there an error/warning when move is performed on const&?

Reply:
Yeah .. It should be a move all the way ..